Resources/public/scss/modules/content-tree/_list.item.scss

```diff
@@ -101,6 +101,8 @@ $list-item-height: calculateRem(20px);
         opacity: 0;
         list-style: none;
         max-height: 0;
+        max-width: 0;
```

Comment:
Note: this fixes a bug - previously list items preserved it's width when collapsed.
Because fix was simple and short I added it in this PR.